templates/blog-post.html

```diff
@@ -27,5 +40,30 @@ <h2 class="mb-0">{{ this.title }}</h2>
       </p>
     </div>
   </div>
+{% if this.series|length %}
+<table class="table table-sm table-striped mt-3" id="series">
+<thead class="thead-dark">
+<tr>
+<th>
+All posts under the Series "<span style="font-size:1.15rem">{{ this.series }}<span>"
+</th>
+</tr>
+</thead>
+<tbody>
+{% for post in series.children %}
```

Comment:
We should style the current post differently in this list, don't have a link to it (since the user is already on the page for the current post).